src/components/settings-menu/DevNotificationsSection.tsx

```diff
@@ -142,7 +146,7 @@ const DevNotificationsSection = () => {
             // @ts-expect-error */}
             <MiniButton
               backgroundColor={colors.blueGreyDark30}
-              color="secondary60"
+              color="dark"
```

Comment:
The original value of `"secondary60"` wasn't being honoured here since `MiniButton` was passing it through to the old `Text` component which falls through to the `buildTextStyles` function which works like this:

```js
const buildTextStyles = css`
  /* Color */
  color: ${({ color, theme }) =>
    colors.get(color, theme.colors) || theme.colors.dark};
```

So as you can see, if you pass in an invalid color name, it falls through to `"dark"`, so I opted to inline the fallback value here and in the other usages of `MiniButton` below.

Reply:
>MiniButton was passing it through to the old Text component which falls through to the buildTextStyles function which resolves color internally like this

😆